pepper-apis/src/main/java/org/broadinstitute/ddp/cache/DaoBuilder.java

```diff
@@ -0,0 +1,8 @@
+package org.broadinstitute.ddp.cache;
+
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+
+public interface DaoBuilder<T extends SqlObject> {
+    T buildDao(Handle handle);
+}
```

Comment:
An attempt to make the new wrapped Daos easily pluggable in place of JDBI DAOs. Examples below.